week5/.gitignore

```
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

```suggestion
node_modules/
```
이렇게 해야 node_moduels가 안올라온다고 합니다. 
그리고 .gitignore는 올라오기 전에 방지해주는거라 이미 올라온 node_modules는 제거해서 다시 올려주셔야합니다 ㅠㅠ

---

아아 넵 알겠습니다